packages/loopring_v3/contracts/impl/libexchange/ExchangeMode.sol

```diff
@@ -117,4 +127,24 @@ library ExchangeMode
             return false;
         }
     }
-}
\ No newline at end of file
+
+    function getNumDowntimeMinutesLeft(
+        ExchangeData.State storage S
+        )
+        internal // inline call
+        view
+        returns (uint)
+    {
+        if (S.downtimeStart == 0) {
+            return S.numDowntimeMinutes;
+        } else {
+            // Calculate how long (in minutes) the exchange is in maintenance
+            uint numDowntimeMinutesUsed = now.sub(S.downtimeStart) / 60;
```

Comment:
use block.timestamp instead of now

Reply:
Any reason for this? `now` is exactly the same as `block.timestamp`:
https://solidity.readthedocs.io/en/develop/units-and-global-variables.html?highlight=block.timestamp#block-and-transaction-properties

All other code uses `now`.

Reply:
Ok, there is a lint warning we should use `block.timestamp` instead of `now`. But if you don't see it, it means we disabled that rule. Then stick to `now` for consistence then.